File: Estrutura_Repetição/Python/Ex1066.py
par = 0
impar = 0
positivos = 0
negativos = 0

for i in range(5):
    n = int(input())
    if n % 2 == 0:
        par += 1
    else:
        impar += 1
    if n > 0:
        positivos += 1
    elif n < 0:
        negativos += 1
print(f"""{par} valor(es) par(es)
{impar} valor(es) impar(es)
{positivos} valor(es) positivo(s)
{negativos} valores(es) negativos(s)""")
par = 0
impar = 0
positivos = 0
negativos = 0

for i in range(5):
    n = int(input())
    if n % 2 == 0:
        par += 1
    else:
        impar += 1
    if n > 0:
        positivos += 1
    elif n < 0:
        negativos += 1
print(f"""{par} valor(es) par(es)
{impar} valor(es) impar(es)
{positivos} valor(es) positivo(s)
{negativos} valor(es) negativo(s)""")

File: Estrutura_Repetição/JavaScript/Ex1066.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

var positivos=[]
var negativos=[]
var pares=[]
var impares=[]

for(i=0;i<5;i++){
    var num = parseFloat(prompt(""))
    if (num > 0){
        positivos.push(num)
    } if (num <0){
        negativos.push(num)
    } if (num%2===0){
        pares.push(num)
    } if (num%2!==0){
        impares.push(num)
    }
}
console.log(`${pares.length} valor(es) par(es)`)
console.log(`${impares.length} valor(es) impar(es)`)
console.log(`${positivos.length} valor(es) positivo(s)`)
console.log(`${negativos.length} valor(es) negativo(s)`)